readme.md

```diff
@@ -369,6 +369,7 @@
 - [PowerShell](https://github.com/janikvonrotz/awesome-powershell) - Cross-platform object-oriented shell.
 - [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows) - Productivity app for macOS.
 - [Terminals Are Sexy](https://github.com/k4m4/terminals-are-sexy)
+- [Monorepos](https://github.com/korfuri/awesome-monorepo) - Monolithic repositories of source code.
```

Comment:
I think you could use a better description here. Not everyone would immediately get it just because it explains what the `mono` part stands for.

https://developer.atlassian.com/blog/2015/10/monorepos-in-git/

Your project is named `Monorepo`, so the title should be changed to that.

Reply:
You could use something from your PR description:

> Short for monolithic repositories, large source-control repositories where multiple projects cohabitate.

?

Reply:
Good idea. I did that (shortened it just a bit for readability).